src/core/diags/factory_diags.cpp

```diff
@@ -72,6 +72,8 @@ namespace FactoryDiags {
 const struct Diags::Command Diags::sCommands[] = {
     {"channel", &Diags::ProcessChannel},
     {"cw", &Diags::ProcessContinuousWave},
+    {"stream", &Diags::ProcessStream},
+    {"stream", &Diags::ProcessStream},
```

Comment:
```suggestion
```
Remove this duplicate

Reply:
Actually, can you shift this down a few lines so it's alphabetically sorted?

Reply:
Make sense. Pushed the changes.